src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -157,12 +154,6 @@ export class ExerciseChatWidgetComponent implements OnInit, OnDestroy, AfterView
                 this.fadeState = 'start';
             }
         });
-
-        // Set initializing flag to false and focus on message textarea
-        setTimeout(() => {
-            this.isInitializing = false;
-            this.messageTextarea.nativeElement.focus();
```

Comment:
I think this is for autofocus, maybe you mis-deleted this

Reply:
right!! thank you, I reverted it